packages/E2ETest/package.json

```diff
@@ -31,8 +31,8 @@
   "devDependencies": {
     "@babel/core": "7.5.5",
     "@babel/runtime": "7.5.5",
-    "@types/react": "16.8.15",
-    "@types/react-native": "~0.60.5",
+    "@types/react-native": "0.61.5",
```

Comment:
It should probably be ~0.61.5, so that dependabot tries to keep it up to date with any future updates to the 61 typings.